contracts/0.8.25/vaults/StakingVault.sol

```diff
@@ -313,16 +320,26 @@ contract StakingVault is IStakingVault, OwnableUpgradeable {
      * @param _deposits Array of deposit structs
      * @dev Includes a check to ensure StakingVault is balanced before making deposits
      */
-    function depositToBeaconChain(Deposit[] calldata _deposits) external {
+    function depositToBeaconChain(
+        Deposit[] calldata _deposits,
+        bytes32 _expectedGlobalDepositRoot,
+        GuardianSignature calldata _signature
```

Comment:
We need just `bytes signature` here and looks like SignatureUtils is not fit for our needs